### PR TITLE
revision_course

### PR DESCRIPTION
**Pull request by iamaffanshahid**

### CODE REVIEW

File: python_revision/if_else_assignment.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    print("F")
+    
+    
+
+    
+        
+    
+    
+        
+
```
Comment (muhammadumerjaved44) [inline python_revision/if_else_assignment.py]: remove these line please

File: python_revision/loops_assignment.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+x = 0
+while x < 3:
+    x += 1
+    for i in my_list:
+        if i == "Monday":
+            print("---------")
+            continue
+        print(i)
+
+
```
Comment (muhammadumerjaved44) [inline python_revision/loops_assignment.py]: remove these extra lines

**Review by muhammadumerjaved44 — CHANGES_REQUESTED** [review]
please do the said changes

File: python_revision/loops_assignment.py (hunk 2 above)
Comment (iamaffanshahid) [reply python_revision/loops_assignment.py]: resolved

File: python_revision/if_else_assignment.py (hunk 1 above)
Comment (iamaffanshahid) [reply python_revision/if_else_assignment.py]: resolved

**Review by muhammadumerjaved44 — APPROVED** [review]
approved

**Pull request merged**